pullrequest.rst

```diff
@@ -63,57 +64,57 @@ Step-by-step Guide
 You should have already :ref:`set up your system <setup>`,
 :ref:`got the source code <checkout>`, and :ref:`built Python <compiling>`.
 
-* Create a new branch in your local clone::
+*  Create a new branch in your local clone::
```

Comment:
What's the reason for this change?

Reply:
Again I'm indenting on 3. And the nearby $ are indented at 6. It was hard to see the on-topic console commands correctly indented when the surrounding list were not.

Reply:
I think the 3-chars indentation only affects the position of the ``*``,  and the rest is aligned with the first char after the ``*`` (so 2 chars).   Similarly if I have a piece of Python code, the code block is indented 3 chars, but within the block the code is indented with 4 characters.
I don't think I ever saw the text of a list indented like this.

Reply:
I recommend trying it if your editor tabs to column 3 for .rst files. But I've reverted what I did here. I've not attempted to fix the indentation generally as part of this PR.